node/node.go

```diff
@@ -128,12 +127,17 @@ func newGethNodeConfig(config *params.NodeConfig) (*node.Config, error) {
 			MaxPeers:        config.MaxPeers,
 			MaxPendingPeers: config.MaxPendingPeers,
 		},
-		IPCPath:          makeIPCPath(config),
+		IPCPath:          config.IPCFile,
 		HTTPCors:         nil,
 		HTTPModules:      config.FormatAPIModules(),
 		HTTPVirtualHosts: []string{"localhost"},
 	}
 
+	if config.IPCEnabled {
+		// resolve relative/absolute path for IPC endpoint
+		nc.IPCPath = nc.IPCEndpoint()
```

Comment:
Isn't it called internally by `node.Node`?

Reply:
oh, right.